creme/creme_core/migrations/0002_historyline_hline__entity_detailview.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1.7 on 2023-03-01 13:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('creme_core', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='historyline',
+            index=models.Index(fields=['entity_id', '-id'], name='hline__entity_detailview'),
+        ),
+    ]
```

Comment:
Rename this file `0120_v2_5__historyline_hline__entity_detailview.py`  (or just "...__historyline_index.py" for example) to follow our naming convention (you can see the previous migrations in `migrations/old/` ).